### PR TITLE
[INTEL MKL] Clean up MKL_ML code for BatchNorm op

### PR DESCRIPTION
**Pull request by gzmkl**
Clean up BatchNorm op by removing legacy MKL-ML code.

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @tatianashp: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by harshini-gadige** [comment]
@tatianashp  PTAL

**Review by tatianashp — DISMISSED** [review]

**Comment by tatianashp** [comment]
Just did. Looks good.

**Comment by gzmkl** [comment]
Hi Tatiana, 
I just made a new commit, to address all "clang format check" related issues (not many).
Hopefully the commit is helpful for the CI test.

Thanks

**Comment by harshini-gadige** [comment]
@tatianashp  Any update ?

**Comment by gzmkl** [comment]
No more update. Since this PR only did code clean up (MKL ML related) and should not introduce 
any error. Only fix is clang style check, adjusting header file inclusion order.

**Comment by harshini-gadige** [comment]
> No more update. Since this PR only did code clean up (MKL ML related) and should not introduce
> any error. Only fix is clang style check, adjusting header file inclusion order.

@gzmkl  Are you working on clang style check ? (Correct me if I understood wrong)

**Review by tatianashp — APPROVED** [review]

**Comment by gzmkl** [comment]
This PR has been blocked by 3 internal checks while most others have passed.

Please help to check. Thanks!

**Comment by tatianashp** [comment]
This PR removes some unused ifdefs. It should be independent from any current test failures.

**Comment by gzmkl** [comment]
Hi Tatiana and your team, thank you very much for taking care of the internal build issue!   -GZ
